room_with_a_view/room_with_a_view.py

```diff
@@ -93,7 +93,7 @@ def parse_args(self):
 
         try:
             with open(self.options.settings, 'r') as stream:
-                settings = yaml.load(stream)
+                settings = yaml.safe_load(stream)
```

Comment:
sanity-check: will the config files ever contain anything other than strings/integers? (If so, we might want to actually specify a Loader instead).

Reply:
It should only contain basic python objects and not python binaries. We only read from two keys in settings -- `connections` and `directories`, and based on the usage it should either be a dictionary or a list. 

Link to all possible objects using safe_load, which calls the SafeLoader created from SafeConstructor
https://github.com/yaml/pyyaml/blob/8cdff2c80573b8be8e8ad28929264a913a63aa33/lib/yaml/constructor.py#L171

Reply:
Makes sense—thank you for the reference!